docs/guides/campaign_recipes/Stacking-Promotion-Tiers.md

```diff
@@ -19,6 +19,9 @@ order: 7
 Cart-level promotions apply discounts to customer's cart without promo codes. Once a customer qualifies for a promotion tier, the discount is automatically added to their cart. 
 Cart promotion can have many tiers, each with different discount and rules assigned. Created tiers can be grouped into stacks and validated/redeemed as a single redeemable. 
 
+> 📘
+> A single promotion stack can have a maximum number of <b>30 tiers</b>.
```

Comment:
To chyba przejdzie w readme, ale zamień proszę na tagi pogrubienia z Markdowna.

Czasem mamy jakieś elementy, które wskazują, że tam w środku leci HTML do zinterpretowania, ale nie wiem, jak w takich sytuacjach.

Dmuchajmy na zimne i trzymajmy się w takich zwykłych miejscach MD.